content/en/docs/reference/glossary/secret.md

```diff
@@ -11,8 +11,17 @@ tags:
 - core-object
 - security
 ---
- Stores sensitive information, such as passwords, OAuth tokens, and ssh keys.
+ Stores sensitive information, such as passwords, OAuth tokens, and SSH keys.
 
 <!--more--> 
 
-Allows for more control over how sensitive information is used and reduces the risk of accidental exposure. Secret values are encoded as base64 strings and stored unencrypted by default, but can be configured to be [encrypted at rest](/docs/tasks/administer-cluster/encrypt-data/#ensure-all-secrets-are-encrypted). A {{< glossary_tooltip text="Pod" term_id="pod" >}} references the secret as a file in a volume mount or by the kubelet pulling images for a pod. Secrets are great for confidential data and [ConfigMaps](/docs/tasks/configure-pod-container/configure-pod-configmap/) for non-confidential data.
+Secrets give you more control over how sensitive information is used and reduces
+the risk of accidental exposure. Secret values are encoded as base64 strings and
+are stored unencrypted by default, but can be configured to be [encrypted at
+rest](/docs/tasks/administer-cluster/encrypt-data/#ensure-all-secrets-are-encrypted).
```

Comment:
```suggestion
are stored unencrypted by default, but can be configured to be
[encrypted at rest](/docs/tasks/administer-cluster/encrypt-data/#ensure-all-secrets-are-encrypted).
```

Reply:
There must be a VS Code formatter setting that I'm missing that'll keep links on one line